lib/mutant/reporter/cli/format.rb

```diff
@@ -42,7 +42,7 @@ class Output
           #
           # @api private
           def tty?
-            @tty
+            tty
```

Comment:
It could. But I do not like it for its inheritance behavior. While I'm not using inheritance here I actually have no reason to avoid it ;)